sergii.holubenko/task_1.rb

```diff
@@ -0,0 +1,17 @@
+arr = Array.new(3){Array.new(3){rand(-10..10)}}.flatten
+
+
+arr.each do |i|
+  if i.positive?
+    puts i
+    summ_elements += i 
```

Comment:
`NoMethodError (undefined method `+' for nil:NilClass)` you haven't defined a variable  `summ_elements`